streamalert/shared/athena.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, database_name, results_bucket, results_prefix, region=None):
         if not results_bucket.startswith('s3://'):
             results_bucket = 's3://{}'.format(results_bucket)
 
-        # Produces athena_partition_refresh/YYYY/MM/DD S3 keys
+        # Produces athena_partitioner/YYYY/MM/DD S3 keys
```

Comment:
This comment seems wrong? It looks like it's supposed to reference the athena query results bucket.

Reply:
yeah I'm unsure where this came from initial .. find/replaced here but will address